atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/transaction/impl/AbstractSerializableTransactionTest.java

```diff
@@ -702,6 +702,25 @@ public void testMultipleReadsToSameColumnRange() {
         t1.commit();
     }
 
+    @Test
+    public void testMultipleReadsToSameColumnRangeAcrossRows() {
+        byte[] row1 = PtBytes.toBytes("row1");
+        byte[] row2 = PtBytes.toBytes("row2");
+
+        Transaction transaction = startTransaction();
+        Map<byte[], BatchingVisitable<Map.Entry<Cell, byte[]>>> columnRange =
```

Comment:
Share the `BatchColumnRangeSelection` to make it clearer it's the same and prevent someone from breaking the test by updating one and not the other.

Reply:
good catch